### ISSUE TITLE
Docs: TOC should come after left-hand navigation and before main content in focus order

### ISSUE THREAD

**Issue by patrickhlauke** [twbs/bootstrap#22849]
Currently the TOC comes after the main content of the page in terms of focus order. For a keyboard user, it means that the TOC which is supposed to help quickly navigate to a section in the page can only be reached after going through the entire page.

Focus order should be as roughly illustrated below (left hand navigation, TOC, main content)

![focus-order](https://user-images.githubusercontent.com/895831/27254166-9d1ebc94-537a-11e7-9880-c9f540a69647.PNG)

**Comment by Mustafa-Online** [comment]
@patrickhlauke Dude, How do u browse docs off-line ?!
.

**Comment by patrickhlauke** [comment]
Dude, you build them from source https://github.com/twbs/bootstrap/blob/v4-dev/docs/4.0/getting-started/build-tools.md ;)

**Comment by mdo** [comment]
Good call! 22851

**Comment by patrickhlauke** [comment]
@mdo just wanted to pop in (late) to verify that #22851 indeed fixes the problem. only outlier at the moment (at least from my testing on Windows) is Firefox <= 54. But Firefox's behavior has been harmonized with what other browsers (including IE11!) do https://bugzilla.mozilla.org/show_bug.cgi?id=812687 and just confirmed it's working fine in Firefox 55 beta. tl;dr: thanks :)
